SoftLayer/managers/vs.py

```diff
@@ -1101,6 +1104,40 @@ def upgrade(self, instance_id, cpus=None, memory=None, nic_speed=None, public=Tr
             return True
         return False
 
+    def get_disk_category_id_by_disk_number(self, capacity, disk_number):
+        """Uses Product_Package::getItemPrices to get all disk items with its categories."""
+        disk_key_names = {
```

Comment:
add information or reference why use this json

Reply:
Done